USAGE.md

```diff
@@ -167,3 +167,20 @@ For additional examples, check out existing manuscripts that use the Manubot (so
 + The Manubot 2018 Development Proposal ([source](https://github.com/greenelab/manufund-2018), [manuscript](https://greenelab.github.io/manufund-2018/))
 
 If you are using the Manubot, feel free to submit a pull request to add your manuscript to the list above.
+
+## Citing Manubot
```

Comment:
I agree it makes sense to add the citation here instead of in the readme to avoid the confusion you described in https://github.com/greenelab/manubot/issues/59